packages/pn-personagiuridica-webapp/src/App.tsx

```diff
@@ -230,9 +230,10 @@ const ActualApp = () => {
   const handleAssistanceClick = () => {
     trackEventByType(TrackEventType.CUSTOMER_CARE_MAILTO, { source: 'postlogin' });
```

Comment:
I wonder whether the TrackEventType shouldn't be different if the user is redirected to `/assistenza` instead of a mail sender.

Reply:
@ZorbaDimatteo please verify that the TrackEventType is right even if the user is logged, thanks!

Reply:
@ZorbaDimatteo if I understood you correctly, the source should be (say) `prelogin` or `postlogin` depending on whether there is a logged user when the link is pressed; I point this since this function is called in both scenarios, logged user or not. Am I right?

Reply:
@clombardi @ImJimMoriarty yes. When the function is called when user is not logged in, Assistenza button will have a mailto to assistance email and the tracking event would have source: "preLogin", otherwise, when user is not logged in, button assistenza will send the user to the new assistenza form and the event source will be "postLogin"

Reply:
OK, now the event' context can be obtained from the source.